### PR TITLE
ChannelSelection Subservice

### PR DESCRIPTION
**Pull request by jbleyel**
[ChannelSelection]
- create, fill and refill a virtual (groupedservices.virtualsubservices.tv) bouquet
- if subservices available, they can be entered via
  - "Reception Lists"
  - ChannelContextMenu new item added
  - Hotkey

[InfoBarGenerics]
- add method for ButtonSetup

[ButtonSetup]
- append "Show subservices"

[EpgSelection]
- provide a subservicelist for all EPG's

### CODE REVIEW

**Comment by IanSav** [comment]
This code causes a boot loop.  This code can not be merged.

File: lib/python/Screens/EpgSelection.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 from Screens.DateTimeInput import EPGJumpTime
 from Screens.EventView import EventViewEPGSelect, EventViewSimple
 from Screens.HelpMenu import HelpableScreen
+from Screens.InfoBar import InfoBar
```
Comment (IanSav) [inline lib/python/Screens/EpgSelection.py:21]: This import causes a boot loop. This code can not be merged.

**Review by IanSav — CHANGES_REQUESTED** [review]
This code can't be merged as it creates a boot loop.  The boot loop makes it impossible to test the code.

I will look at the code and try to create a working version.

**Comment by GhostofGeeeee** [comment]
Looks feasible, but it's not my branch...

**Comment by jbleyel** [comment]
> Looks feasible, but it's not my branch...

It's your commit :

https://github.com/GhostofGeeeee/enigma2/commit/3a86d5c6cdb22974908b3ae468658c8815556b76#diff-70806cbca1d82e0e3d304147ae4ecb5ea53bfc085062721bf79ee5c954ddf532

;-)

**Comment by GhostofGeeeee** [comment]
This is not my PR.

**Comment by jbleyel** [comment]
> This is not my PR.

Yes it is.

https://github.com/openatv/enigma2/pull/3265

**Comment by GhostofGeeeee** [comment]
And how to fix a commit or a closed PR?

**Comment by jbleyel** [comment]
We are currently working on this task.

**Pull request merged**